### DISCUSSION TITLE
how to automatically redeploy/sync job/pod when ConfigMap changes?

### DISCUSSION THREAD

**Discussion by tooptoop4** [argoproj/argo-cd#8555] [Q&A]
how to automatically redeploy/sync job/pod when ConfigMap changes?

**Comment by steinarox** [comment]
Not an argo specific question but, helm can do it like this:
https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments

If you dont use helm you can manually create an annotation/label that you change when you update the configmap
